cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/service/BusinessMetadataStore.java

```diff
@@ -97,6 +99,21 @@ public Void apply(BusinessMdsIterable input) throws Exception {
   }
 
   /**
+   * @return a {@link MetadataRecord} representing all the metadata (including properties and tags) for the specified
+   * {@link Id.NamespacedId}.
+   */
+  public Set<MetadataRecord> getMetadata(final Id.NamespacedId entityId) {
```

Comment:
Ignore the previous comment. The set is to contain both user and system level ``MetadataRecord``. ``BusinessMetadataStore`` should only return user scope metadata. ``DefaultMetadataAdmin`` needs possibly make calls to multiple stores in future to get both user and system metadata.